setup.py

```diff
@@ -5,25 +5,21 @@
 from distutils.core import Extension
 import platform
 
-INSTALL_REQUIRES = [
-    'futures;python_version<"3.2"',
-    'enum34;python_version<"3.4"',
-]
+work_dir = os.path.dirname(os.path.realpath(__file__))
+src_dir = os.path.join(work_dir, 'confluent_kafka')
+mod_dir = os.path.join(src_dir, 'src')
```

Comment:
I think these are reversed.
The python module base dir is confluent_kafka, while the source dir is confluent_kafka/src, so maybe flip them?

Reply:
(work_dir) repo base "confluent_kafka" 
(src_dir) package base "./confluent_kafka"
(src_dir) python module base "./confluent_kafka"
(mod_dir) module extension base "./confluent_kafka/src"

What if we rename it ext_dir  and rename src_dir to mod_dir